app/pb_kits/playbook/pb_home_address_street/_home_address_street.jsx

```diff
@@ -0,0 +1,64 @@
+/* @flow */
+/*eslint-disable react/no-multi-comp, flowtype/space-before-type-colon */
+
+import React from 'react'
+
+import classnames from 'classnames'
+
+import Body from '../pb_body/_body.jsx'
+import Title from '../pb_title/_title.jsx'
+
+type HomeAddressStreetProps = {
+  address?: String,
+  city?: String,
+  className?: String,
+  dark?: Boolean,
+  homeId?: Number,
+  houseStyle?: String,
+  state?: String,
+  zipcode?: String,
```

Comment:
I think we would only want to make these props optional: 

- `dark`
- `className`